templates/company/view_all/view.html.tx

```diff
@@ -74,5 +74,29 @@
         <a class="govuk-button secondary-button piwik-event" data-event-id="Order a certificate" href="orderable/certificates">Order certificate</a>
     % }
   %}
+
+  % if $c.config.feature.order_certified_document {
+    % if $show_certified_document {
+      <h2 id="order-a-certified-document-header" class="heading-large heading-with-border">
+        Order a certified document
+      </h2>
+      <details class="govuk-details" role="group">
+        <summary class="govuk-details__summary" role="button" aria-controls="details-content" aria-expanded="true">
+          <span class="govuk-details__summary-text">
+            What you'll need
+          </span>
+        </summary>
+        <div class="govuk-details__text" id="order-certified-document-details-content" aria-hidden="false">
+          <p id="orders-list">To use the service, you'll need:</p>
```

Comment:
This `id` is a duplicate and could be named: `order-certified-list`

Reply:
Updated the id